app/my_jss/accounts/Users.js

```diff
@@ -0,0 +1,79 @@
+var jsonUsers = JSON.parse(usersJSON);
+
+var userData = new DataController();
+userData.setUrlLocation(usersJSON);
+userData.setSaveName("users");
+
+function Users(){
+
+ this.allUsers = this.getAllUsers();
+}
+
+//set user state ////
+
+Users.prototype.setUserState = function(email,state){
+
+  var localUsers = this.allUsers;
+
+  for(var i=0; i < localUsers.length; i++){
+
+      if(localUsers.email === email)
+      {
+         localUsers.setLoginState(state);
+      }
+    }
+};
+// returns boolean
+Users.prototype.checkedByType = function(type,objectKey){
+
+
+  var localUsers = this.allUsers;
+   for(var i=0; i < localUsers.length; i++){
+
+      var user = localUsers[i];
+
+     if(user[type] === objectKey)
+      return true;
+   }
+ return false;
+};
+
+// returns array of all users//
+Users.prototype.getAllUsers = function () {
+
+  var thisOfUsers = this;
+
+  var mapFunction = function(val){
+
+      return  thisOfUsers.createUser(val.name, val.email, val.password);
```

Comment:
Very good. createUser method i find it unnecesary because it doesn't provide any value added. just do a return new User(...). Same for Song